database/index.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const Soundtrack = require('./models/soundtracks.js');
+
+mongoose.connect('mongodb://localhost/bonuses', {
+  autoIndex: false,
+  dbName: 'bonuses',
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}).then( () => {
+  console.log('Connection to MongoDB successful!');
+}).catch( (err) => {
+  console.log('DB connection error:', err);
+});
+
+// Get bonus associated with bundle
+const getBonuses = (bundleId, callback) => {
+  Soundtrack.find({ bundle_id: bundleId }, (err, data) => {
+    if (err) {
+      err = new Error(err);
+      callback(err);
+    } else {
+      callback(null, data);
+    }
+  });
+}
+
+// Get individual bonus item info
+const getBonusItem = (title, callback) => {
+  Soundtrack.find({ "bonus_info.title": `${title}` }, (err, data) => {
```

Comment:
I read this as invoking this `find` function on a key called `bonus_info.title`, does mongoose expect you to enter in key names in double quotes? I believe we're supposed to be using single quotes where possible. I also am not sure that you need the string literal around the value (see your use of `bundleId` without the string literal on line 17).